csfieldguide/chapters/content/en/complexity-and-tractability/sections/the-travelling-salesman-problem.md

```diff
@@ -100,15 +100,15 @@ Just like in that activity you listed all the possible ordering for the values i
 How many possible routes are there for the larger example you have generated?
 How is this related to permutation sort, and factorials?
 How long would it take to calculate the shortest route in your map, assuming the computer can check 1 billion (1,000,000,000) possible routes per second?
-(i.e. it can check one route per nanosecond) What can you conclude about the cost of this algorithm?
+(i.e. it can check one route per nanosecond). What can you conclude about the cost of this algorithm?
```

Comment:
I removed '(i.e. it can check one route per nanosecond)'. I thought it was unnecessary as we already stated it can check 1 billion routes per second. Thoughts?

Reply:
I also kept 1,000,000,000 as I feel it's good to have as a copy-paste for the interactive

Reply:
That also works fine, I approve